docs/scos/dev/front-end-development/202212.0/oryx/getting-started/oryx-environment.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Environment Variables"
+---
+
+Environment variables are useful when:
+
+- Values are different across developer machines.
+- Values are different across multiple environments: (dev, staging, qa, prod)
+- Values change frequently and are highly dynamic.
+
+Environment variables can be changed easily - especially when running in CI.
+
+Oryx uses [dotenv](https://github.com/motdotla/dotenv) for environment variables management. 
```

Comment:
Oryx is not concerned much with env vars other then loading them. What is dotenv given us? Aren't we just using Vite?

Reply:
Changed to vite

Reply:
My bad on this one, it's slightly different. You were right on dotenv. Vite is using dotenv as well. 

However, we could leave it out, as it's an unrelevant detail maybe.

@tolerants what do you think? Is it relevant if folks move to alternative build system?

Reply:
I'd leave an initial version, so anyone is using dotenv can get an idea. And maybe we can can add a note that it can be easily replaced by customer.

> Is it relevant if folks move to alternative build system?
Sure it will be still relevant for at least some part of our audience, for example parcel is also using dotenv.

Reply:
I've replaced it with `Oryx uses vite [.env files](https://vitejs.dev/guide/env-and-mode.html#env-files) for environment variables management.` so should be fine, right?